files/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.autoSave": "on",
+  "editor.tabSize": 2,
```

Comment:
@jelaniwoods Ahhh thank goodness for this, I noticed it and was wondering why it started.